ui/packages/consul-ui/app/templates/dc/acls/auth-methods/show/namespace-rules.hbs

```diff
@@ -0,0 +1,21 @@
+
+<div class="tab-section">
+{{#if (gt item.NamespaceRules.length 0)}}
+  <p>A set of rules that can control which namespace tokens created via this auth method will be created within. Unlike binding rules, the first matching namespace rule wins.</p>
```

Comment:
I think this `<p>` tag needs to show whether you have rules or not, check with design maybe but in other places in the UI we still show the info paragraphs at the top if there are no results to show.